h/templates/activity/search.html.jinja2

```diff
@@ -143,7 +143,7 @@
 {% macro tags_section() %}
 <section class="search-result-sidebar__section">
   <h3 class="search-result-sidebar__subtitle">
-    {% trans %}Top tags{% endtrans %}
+    {% trans %}Tags{% endtrans %}
```

Comment:
I see this change was part of the design, but there is some context of which @dawariley may not have been aware. The tag list does not show _all_ tags but currently only the top N tags (ie. most frequently used in annotations in the group), so "Top tags" is a more accurate description.

I would prefer that we leave this change out of this PR and address separately.

Reply:
Ah, ok I was looking at the code and didn't see where it was limiting the tags?

Reply:
Oh, don't worry, it is completely non-obvious! The tags come from an Elasticsearch feature called [aggregations](https://www.elastic.co/guide/en/elasticsearch/reference/1.5/search-aggregations.html) and are configured [here](https://github.com/hypothesis/h/blob/db7f44e1432ef2c065d97aa7a2c4b9483c3d2bdd/h/activity/query.py#L145).